sources/packages/web/src/composables/useFormatters.ts

```diff
@@ -210,19 +210,26 @@ export function useFormatters() {
   };
 
   /**
-   * Format a currency value.
+   * Format a currency value including negative currency value.
    * @param moneyAmount value amount.
    * @param currencyCode currency code.
+   * @param defaultValueAmount default value to display
+   * when value is not provided or zero.
    * @returns formatted currency value.
    */
   const formatCurrency = (
     moneyAmount: number | string,
     currencyCode = "(CAD)",
-  ): string | null => {
+    defaultValueAmount = "$0.00",
```

Comment:
I would prefer to have 0(zero) assigned to the `moneyAmount` and let the format be done by the method.
The consumer of the method can also do it using the `??` for the cases where it makes sense.